CONTRIBUTING.md

```diff
@@ -104,19 +117,17 @@ Tags disponíveis:
 - `Design`;
 - `Ferramenta`;
 - `Framework`;
+- `Front-End`;
 - `Front-end`;

```

Comment:
Aqui acho que vale a gente padronizar né? Essa questão das tags serem manuais dificulta muito nossa vida 😵‍💫

Reply:
Por enquanto botei tudo só com a 1a letra minúscula (é como tá nos cards). Depois a gente pode pensar se mantém ou não essas tags nesse arquivo. Porque a pessoa que for contribuir pode só abrir o site e clicar no filtro de tags pra ver quais tem, enquanto manter aqui atualizado vai ser um pouco mais chato.